kubetest/anywhere.go

```diff
@@ -65,8 +69,10 @@ const kubernetesAnywhereConfigTemplate = `
 .phase2.docker_registry="gcr.io/google-containers"
 .phase2.kubernetes_version="{{.KubernetesVersion}}"
 .phase2.provider="{{.Phase2Provider}}"
-.phase2.kubeadm.version="{{.KubeadmVersion}}"
+.phase2.upgrade_provider="{{.UpgradeProvider}}"
```

Comment:
There is only one new config value on the kubenetes anywhere end: ".phase2.kubeadm.master_upgrade.method"
 
It behaves as documented in the [approach doc](https://docs.google.com/a/google.com/document/d/1PoDeqD8qu1KbAd-fb7xOVB1G7YTyD6VLLlhaVB3QeA4/edit?usp=sharing).